assets/queries/openAPI/security_empty_array/query.rego

```diff
@@ -0,0 +1,33 @@
+package Cx
+
+import data.generic.openapi as openAPILib
+
+CxPolicy [ result ] {
+  doc := input.document[i]
+  openAPILib.checkOpenAPI(doc) != "undefined"
+  object.get(doc, "security", "undefined") == "undefined"
+
+	result := {
+                "documentId": 		doc.id,
+                "searchKey": 	    "openapi",
+                "issueType":		"MissingAttribute",
+                "keyExpectedValue": "A default security schema should be defined",
+                "keyActualValue": 	"A default security schema is not defined"
+              }
+}
+
+CxPolicy [ result ] {
+  doc := input.document[i]
+  openAPILib.checkOpenAPI(doc) != "undefined"
+  object.get(doc, "security", "undefined") != "undefined"
+
+  count(doc.security) == 0
+
+	result := {
```

Comment:
Use [vscode-opa](https://github.com/open-policy-agent/vscode-opa) extension and enable format on save under settings to format your Rego code.